src/plugins/data_view_management/public/components/edit_index_pattern/index_header/index_header.tsx

```diff
@@ -44,14 +46,32 @@ const removeTooltip = i18n.translate('indexPatternManagement.editDataView.remove
   defaultMessage: 'Delete',
 });
 
+const refreshAriaLabel = i18n.translate('indexPatternManagement.editDataView.refreshAria', {
+  defaultMessage: 'Refresh',
+});
+
+const refreshTooltip = i18n.translate('indexPatternManagement.editDataView.refreshTooltip', {
+  defaultMessage: 'Refresh local copy of data view field list',
```

Comment:
The tooltip stays visible and stuck after pressing the button. Do it hide for you?